nodejs/index.js

```diff
@@ -0,0 +1,96 @@
+const https = require('https');
+const querystring = require('querystring');
+const aws4 = require('./asv4');
```

Comment:
If this file is supposed to be named asv4 then it might be confusing to rename it "aws4" when you are importing it.

Reply:
Good catch. I renamed aws4 to asv4